udata_front/api.py

```diff
@@ -77,11 +77,14 @@ def get(self):
         if args['get'] == "p":
             return json.loads(req.content)
         request.accept_mimetypes = MIMEAccept(accept)
-        return req.content
 
-    @apiv2.representation('image/*')
-    @apiv2.representation('audio/*')
-    def media(data, code, headers):
-        resp = make_response(bytes(data), code)
-        resp.headers.extend(headers)
-        return resp
+        values_gen = list(request.accept_mimetypes.values())
```

Comment:
Not sure we need to meddle with `request.accept_mimetypes` anymore.
Could we directly use `make_response` based on `args['get']` being image or audio?